src/System.Private.CoreLib/src/System/Random.cs

```diff
@@ -126,6 +132,35 @@ private int InternalSample()
             return retVal;
         }
 
+        [ThreadStatic]
+        private static Random t_threadRandom;
+        private static readonly Random s_globalRandom = new Random(GenerateGlobalSeed());
+
+        /*=====================================GenerateSeed=====================================
+        **Returns: An integer that can be used as seed values for consecutively
+                  creating lots of instances on the same thread within a short period of time.
+        ========================================================================================*/
+        private static int GenerateSeed() {
```

Comment:
Nit - the CoreRT code was reformatted to follow [corefx coding conventions](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md). `{` should be on a new line.